docs/book/src/tasks/kubeadm-control-plane.md

```diff
@@ -16,7 +16,24 @@ See the section on [upgrading clusters][upgrades].
 
 See the section on [Adopting existing machines into KubeadmControlPlane management][adoption]
 
+### Running workloads on control plane machines
+
+The user may want to run non-control plane workloads on control plane machines.
+
+In this case, the user is also the ultimate responsible for ensuring the
+proper functioning of those workloads, given that KCP is not aware of the specific
+requirements for each type of workload (e.g. preserving quorum, shutdown procedures etc.).
+
+In order to do so, the user could leverage on the same assumption that applies to all the
+Cluster API Machines:
+
+- The Kubernetes node hosted on the Machine will be cordoned & drained before removal (with well
+  know exceptions like full Cluster deletion).
+- The Machine will respect PreDrainDeleteHook and PreTerminateDeleteHook. see the
```

Comment:
Again a minor nit, camel casing is missing in "see the Machine Deletion Phase Hooks proposal"